src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/message/ReclusterService.kt

```diff
@@ -43,6 +41,7 @@ class ReclusterService(
     }
   }
 
+  @Transactional
```

Comment:
as above

Reply:
this annotation needs moving to the class or a public method, or removing altogether